docs/reference/run.mdx

```diff
@@ -147,29 +147,29 @@ The filter syntax supports the following operations:
 **Equal filter**: To filter events where the `comm` field equals `cat`, use:
 
 ```bash
---filter comm==cat
+--filter proc.comm==cat
 ```
 
 This filters for events related to the `cat` command.
 
 **Not equal filter**: To filter out events where the `uid` is not 0 (root user), use:
 
 ```bash
---filter uid!=0
+--filter proc.creds.uid!=0
 ```
 
 **Regular expression filter**: To match a regular expression in a specific column, use:
 
 ```bash
---filter comm~'^c.*'
+--filter proc.comm~'^c.*'
```

Comment:
```suggestion
--filter 'proc.comm~^c.*'
```

The other examples could have simple quotes too.

Reply:
Yeah, quotes are necessary when using negative regex like `-F 'proc.comm!~^c.*'`. In any case, I think it's a good practice to suggest using quotes. WDYT? If you agree, I suggest adding it to the documentation.

Reply:
I agree. Fixed.

Reply:
@blanquicet I'm not clear on your other suggestion. Could you clarify what I should change compared to the current state of the PR?